scss/spinners/2-double-bounce.scss

```diff
@@ -13,12 +13,11 @@
   &.sk-spinner {
```

Comment:
We use this pattern throughout SpinKit, but I'm considering if we should just kill this selector and make these styles part of the parent selector. E.g. making this:

```css
.sk-spinner-double-bounce {
   [styles]
}
```
…rather than:
```css
.sk-spinner-double-bounce {
   &.sk-spinner {		
      [styles]
   }
}
```

Thoughts? :heart: